pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -550,11 +549,11 @@ func (ctx *localScopeInstallContext) runPlaybook() (string, string, error) {
 		Stdin: strings.NewReader(command),
 	}
 
-	stdOut, stdErr, err := splutil.PodExecCommand(*worker.client, worker.targetPodName, cr.GetNamespace(), []string{"/bin/sh"}, streamOptions, false, false)
+	stdOut, stdErr, err := splutil.PodExecCommand(worker.client, worker.targetPodName, cr.GetNamespace(), []string{"/bin/sh"}, streamOptions, false, false)
```

Comment:
Yes, this is a TBD item from the past PRs. Also, we have other places in the code, from other features. This is really something we need to address across the code, once we have the podExecClient strengthened to mimic some statemachine logic, that cane gives us combinations of return values so that we can drive the function flow when multiple podExec commands are involved.

Reply:
Looking at the overall flow seems like we also need better introduce the podExec client to the worker, that way, it can be re-used across the podCopy, extraction, install, deletion etc. However, in order to not to duplicate, we can have a pointer reference across the workers, with the deviation and extra logic for the Standalone case :-).